teamscale-maven-plugin/pom.xml

```diff
@@ -0,0 +1,170 @@
+<project>
+    <modelVersion>4.0.0</modelVersion>
+
+    <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <revision>1.0.0-SNAPSHOT</revision>
+        <teamscale.agent.version>23.1.1</teamscale.agent.version>
+    </properties>
+
+    <groupId>com.teamscale</groupId>
+    <artifactId>teamscale-maven-plugin</artifactId>
+    <!--
+        "revision" is a special name that prevents warnings on the console
+        c.f. https://maven.apache.org/docs/3.2.1/release-notes.html
+    -->
+    <version>${revision}</version>
+    <packaging>maven-plugin</packaging>
+
+    <name>Teamscale Maven Plugin</name>
+
+    <licenses>
+        <license>
+            <name>The Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+    <scm>
+        <connection>scm:git:git@github.com:cqse/teamscale-jacoco-agent.git</connection>
+        <developerConnection>scm:git:ssh://github.com:cqse/teamscale-jacoco-agent.git</developerConnection>
+        <url>https://github.com/cqse/teamscale-jacoco-agent</url>
+    </scm>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>5.8.2</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.teamscale</groupId>
+            <artifactId>teamscale-jacoco-agent</artifactId>
+            <version>${teamscale.agent.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-plugin-api</artifactId>
+            <version>3.8.4</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven.plugin-tools</groupId>
+            <artifactId>maven-plugin-annotations</artifactId>
+            <version>3.6.4</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-core</artifactId>
+            <version>3.8.5</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.jgit</groupId>
+            <artifactId>org.eclipse.jgit</artifactId>
+            <version>5.11.1.202105131744-r</version>
+        </dependency>
+        <dependency>
+            <groupId>com.teamscale</groupId>
+            <artifactId>teamscale-lib-commons</artifactId>
+            <version>7.7.5</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
```

Comment:
I think it is worth adding a `<pluginManagement>` configuration for the `maven-wrapper-plugin` https://maven.apache.org/wrapper/maven-wrapper-plugin/wrapper-mojo.html#distributionType setting, to configure   `script`.